platforms/documentation/docs/src/snippets/buildCache/configure-built-in-caches/groovy/settings.gradle

```
@@ -4,6 +4,8 @@ rootProject.name = 'configure-built-in-caches'
 buildCache {
     local {
         directory = new File(rootDir, 'build-cache')
+        // enabled = false // Uncomment to disable the local build cache
+        // push = false // Uncomment to disable storing outputs in the build cache
```

```suggestion
        // push = false // Uncomment to disable storing outputs in the local build cache
```

Just to clarify that this affects only local build cache

---

Fixed.